### PR TITLE
Update Lighthouse page

### PR DESCRIPTION
**Pull request by ebidel**
R: @kaycebasques @petele 

This PR updates https://developers.google.com/web/tools/lighthouse/ with newer screenshots, adds and install and file a bug call to action, and adds new section on updated LH features.
It also updates the /tools landing page to raise LH above the fold.

cc @brendankenny @paulirish @patrickhlauke 

screen shot 2017-01-18 at 10 09 46 pm

screen shot 2017-01-18 at 10 09 51 pm

### CODE REVIEW

**Comment by PaulKinlan** [comment]
It's missing the song.

File: gae/article.tpl [hunk 1]
```diff
@@ -5,9 +5,10 @@
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,400italic,500,500italic,700,700italic|Roboto+Mono:400,500,700|Material+Icons">
     <link rel="stylesheet" href="/wf-local/styles/devsite-google-blue.css">
     <meta name="robots" content="noindex">
+    <link rel="chrome-webstore-item" href="https://chrome.google.com/webstore/detail/blipmdconlkpinefehnmjammfjpmpbjk">
```
Comment (petele) [inline gae/article.tpl]: This file is only used for our staging environment - do you really want to add this here as well? Just confirming.

File: src/content/en/tools/lighthouse/index.md [hunk 2]
```diff
@@ -1,84 +1,175 @@
...
+To install the Node module, do the following:
+
+1. Install [Node](https://nodejs.org), version 5 or greater.
+- Install Lighthouse as a global module:
+
+        npm install -g lighthouse
+
+Then, run Lighthouse against a URL to audit the page:
+
+    lighthouse https://airhorner.com/
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/index.md]: Can I assume that I can `lighthouse localhost:8000/`?

File: src/content/en/tools/lighthouse/index.md [hunk 3]
```diff
@@ -1,84 +1,175 @@
...
+There are several ways to view a report:
 
-Run a Lighthouse audit against a page.
+1. When viewing the HTML version of a report, click the "Open in viewer" button
+in the top right corner of the report.
+2.  Drag and drop the JSON output of a Lighthouse run onto the app or click to
+upload a report file. If you're a CLI user, run Lighthouse with the `--output=json` flag to
+generate a json file that can be opened in the Viewer.
+3. Add the `?gist=<GISTID>"` parameter to the URL of the Viewer.
+4. Paste the URL of a Github gist that contains a Lighthouse json report onto the app.
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/index.md]: I suggest linking to https://gist.github.com/

File: gae/article.tpl (hunk 1 above)
Comment (ebidel) [inline gae/article.tpl]: done

File: src/content/en/tools/lighthouse/index.md (hunk 2 above)
Comment (ebidel) [inline src/content/en/tools/lighthouse/index.md]: Yes. Any URL that's accessible via Chrome.

File: src/content/en/tools/lighthouse/index.md (hunk 3 above)
Comment (ebidel) [inline src/content/en/tools/lighthouse/index.md]: done

**Comment by ebidel** [comment]
Merge ready?

**Comment by kaycebasques** [comment]
@ebidel no

...j/k

**Comment by petele** [comment]
Staged - will go live in the next couple of hours.

**Pull request merged**